packages/node-sdk/example/app.ts

```diff
@@ -0,0 +1,73 @@
+import express from "express";
+import { bucket } from "./bucket";
+import { BucketClient } from "../src";
+import type { AppFlags } from "./flags";
+
+export const app = express();
+
+declare global {
+  namespace Express {
+    interface Locals {
+      bucketUser: BucketClient;
+    }
+  }
+}
+
+app.use(express.json());
+app.use((req, res, next) => {
+  const bucketUser = bucket
+    .withUser(req.headers["x-bucket-user-id"] as string, {
+      attributes: {
+        role: req.headers["x-bucket-is-admin"] ? "admin" : "user",
+      },
+    })
+    .withCompany(req.headers["x-bucket-company-id"] as string, {
+      attributes: {
+        betaUser: !!req.headers["x-bucket-company-beta-user"],
+      },
+    });
+
+  res.locals.bucketUser = bucketUser;
```

Comment:
Hmm, not sure what's going on here :/ 

<img width="1570" alt="Screenshot 2024-07-22 at 11 14 38" src="https://github.com/user-attachments/assets/0e8ddb9d-15ec-4e0a-87fe-183dec9e87f6">